src/lib/spinel/radio_spinel.cpp

```diff
@@ -51,6 +51,35 @@
 namespace ot {
 namespace Spinel {
 
+char RadioSpinel::sVersion[kVersionStringSize] = "";
+
+otExtAddress RadioSpinel::sIeeeEui64;
+
+bool RadioSpinel::sIsReady = false; ///< NCP ready.
+
+bool RadioSpinel::sSupportsLogStream =
+    false; ///< RCP supports `LOG_STREAM` property with OpenThread log meta-data format.
```

Comment:
Can this be different from each IID?

Reply:
I don't think it can. STREAM_LOG uses OPENTHREAD_SPINEL_CONFIG_BROADCAST_IID for now. Current logging system implementation does not take instance into account.

Reply:
👍